packages/wallet/frontend/src/components/TransferHeader.tsx

```diff
@@ -17,18 +16,12 @@ type TransferHeaderProps = {
 export const TransferHeader = ({ type, balance }: TransferHeaderProps) => {
   return (
     <div className="text-center">
-      <div className="relative mb-16 mt-7 flex items-center">
-        <BackButton />
-        <div className="absolute left-[50%] -translate-x-1/2">
-          <h2 className={cx('text-xl font-light', TYPES[type])}>
-            Total balance
-          </h2>
-          {balance && (
-            <h3 className={cx('h-10 text-3xl font-semibold', TYPES[type])}>
-              {balance}
-            </h3>
-          )}
-        </div>
+      <div className="mb-16 mt-7 flex flex-col items-center justify-center">
+        <h2 className={cx('text-xl font-light', TYPES[type])}>Total balance</h2>
+
+        <h3 className={cx('h-10 text-3xl font-semibold', TYPES[type])}>
+          {balance ? balance : '-.--'}
+        </h3>
```

Comment:
We need to keep the `BackButton` component for mobile.

Reply:
@raducristianpopa PageHeader has it, if we keep it in TransferHeader also, it will be duplicated